packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -46,6 +53,7 @@ type PersonalizeQueryResult = {
 
 export class GraphQLPersonalizeService {
   private graphQLClient: GraphQLClient;
+  private memoryCache: CacheClient<PersonalizeQueryResult>;
```

Comment:
Not necessarily a memory cache (i.e. if overridden). I'd rename.
```suggestion
  private cache: CacheClient<PersonalizeQueryResult>;
```